onnxruntime/test/onnx/OrtValueList.h

```diff
@@ -4,6 +4,8 @@
 #pragma once
 #include "core/session/onnxruntime_cxx_api.h"
 
+extern const OrtApi* g_ort;
```

Comment:
Not sure, I like this g_ declarations within so many headers. Don't we already have one in the above header?

Reply:
The C++ API defines one in its own namespace but the C API doesn't. Since the C++ API uses this directly I had to name it. The C API never refers to this symbol itself, so it's all open to the client users.